### PR TITLE
feat: Add gray-lighter to DS colors

### PR DESCRIPTION
**Pull request by alexcastelo95**

### CODE REVIEW

File: src/scss/variables/_colors.scss [hunk 1]
```diff
@@ -34,6 +34,7 @@ $colors: (
   ),
   gray: (
     light: #dcdcdc,
+    lighter: #f6f6f6,
```
Comment (edufarre) [inline src/scss/variables/_colors.scss:37]:
```suggestion
    lighter: #f6f6f6,
    light: #dcdcdc,
```

**Review by edufarre — APPROVED** [review]

File: src/scss/variables/_colors.scss (hunk 1 above)
Comment (afontcu) [reply src/scss/variables/_colors.scss:37]: toc toc toc

Comment (edufarre) [reply src/scss/variables/_colors.scss:37]: image

**Comment by afontcu** [comment]
Note that this PR should issue a new release, so the merge commit must start with `feat: `. You could do so by modifying the commit after clicking Squash and merge, or simply by editing the PR title (because the default squash commit takes its value from the title of the PR)

**Review by afontcu — APPROVED** [review]

**Pull request merged**